packages/story-editor/src/utils/copyPaste.js

```diff
@@ -149,5 +146,3 @@ export function addElementsToClipboard(page, elements, animations, evt) {
     `<!-- ${serializedPayload} -->${htmlContent}`
   );
 }
-
-export { getOffsetCoordinates as getPastedCoordinates };
```

Comment:
The changes in this file here are unrelated, but webpack complained about it when I started debugging Fast Refresh.

This re-export was completely unused and doesn't make sense anyway.